bin/filter.rb

```diff
@@ -24,6 +24,13 @@ def process(data, force_header: nil, format_for_frontpage: false)
 
   doc = Nokogiri::HTML::DocumentFragment.parse(body)
 
+  # Grab footer image for social media previews
+  if !(header =~ /image:/i)
+    image = doc.css("img.mcnImage").last
+    image_url = image['src']
+    header = header + "image: #{image_url}\n"
```

Comment:
Assuming header ends with a `\n`, this seems reasonable.

Reply:
it does because the trailing `---` has to occur on its own line.